server/server.go

```diff
@@ -101,6 +101,8 @@ var (
 	etcdCommittedIndexGauge = etcdStateGauge.WithLabelValues("committedIndex")
 )
 
+var _ tso.Member = (*member.EmbeddedEtcdMember)(nil)
```

Comment:
Why do we need to make sure `member.EmbeddedEtcdMember` could satisfy `tso.Member` here?

Reply:
I think we can remove this line.

Reply:
removed